.env.example

```diff
@@ -50,5 +50,5 @@ TWILIO_MESSAGE_VALIDITY_PERIOD=
 DST_REFERENCE_TIMEZONE='America/New_York'
 PASSPORT_STRATEGY=local
 EXPERIMENTAL_TAGS=1
-TEXTER_SIDEBOXES=celebration-gif,default-dynamicassignment,default-releasecontacts,contact-reference,tag-contact
+TEXTER_SIDEBOXES=celebration-gif,default-dynamicassignment,default-releasecontacts,contact-reference,tag-contact,freshworks-widget
```

Comment:
Small nit: can you change it to `freshdesk-widget`? That's it's real name and I want to make sure admin can easily find it.

Reply:
ah sorry I didn't realize the company is freshworks! Ignore please!